docs/contributing/coding_guide.rst

```diff
@@ -85,21 +85,9 @@ Coding guidelines
 
   .. code-block:: pycon
 
-     >>> [x ** 2 for x in range(17) if x % 2 == 0]
+     >>> [x**2 for x in range(17) if x % 2 == 0]
      [0, 4, 16, 36, 64, 100, 144, 196, 256]
 
-  A comprehension might be more readable when spread out over multiple
-  lines.
-
-  .. code-block:: pycon
-
-     >>> {
-     ...     x: x ** 2
-     ...     for x in range(17)
-     ...     if x % 2 == 0
-     ... }
-     {0: 0, 2: 4, 4: 16, 6: 36, 8: 64, 10: 100, 12: 144, 14: 196, 16: 256}
-
```

Comment:
This isn't relevant to us anymore since this sort of formatting is handed by autoformatters.  Hence, we can delete it!